src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { initServer } from "./app/app.server";
+import "dotenv/config";
+
+initServer(+process.env.PORT || 666);
```

Comment:
jajajaja....nice try with the port 😂

Reply:
If it works, it works, here, a little demonic emoticon for your comment:

c:[